### PR TITLE
Create new /download/cloud/build-openstack page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Created the new /download/cloud/build-openstack page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/download/cloud/build-openstack](http://0.0.0.0:8001/download/cloud/build-openstack)
- Check that the copy matches the [copy doc](https://docs.google.com/document/d/1b-KJ3Ext5geHtRU26UrXusLNoulSJ3jqvLl5NlwbyQo/edit#heading=h.wc7vrfwrcrio)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2181

### CODE REVIEW

File: templates/download/cloud/build-openstack.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  <div class="row">
+    <div class="col-10">
+      <h1>Build your own production cloud</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="p-card--highlighted">
+      <div class="u-equal-height p-divider">
+        <div class="col-6 p-divider__block">
+            <h2>OpenStack with conjure-up</h2>
```
Comment (deadlight) [inline templates/download/cloud/build-openstack.html]: Indentation off

File: templates/download/cloud/build-openstack.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip--light is-deep" id="instructions">
+  <div class="row">
+    <div class="col-12">
+      <h2>Installation instructions</h2>
+
```
Comment (deadlight) [inline templates/download/cloud/build-openstack.html]: remove new lines

**Comment by deadlight** [comment]
Some text too close to the next column:

build-openstack

**Comment by kwm14** [comment]
## Visual review

### Installation instructions

**6. Install and launch conjure-up**
- [ ] Bug: on hover the code block reveals full command line
screen shot 2017-09-12 at 17 21 24

**7. Select install type**
- [ ] Capitalise first word 'Conjure-up' to match copy doc

**10. Configure, architect and deploy the OpenStack services**
- [ ] Padding issue on 6-col layout
screen shot 2017-09-12 at 17 30 11

**Contextual footer**
- [ ] Content in second block 'Growing your cloud?' doesn't match what is seen in the copy doc [here](https://docs.google.com/document/d/1b-KJ3Ext5geHtRU26UrXusLNoulSJ3jqvLl5NlwbyQo/edit#heading=h.gt89bdtcmhos)

**Comment by anthonydillon** [comment]
> Some text too close to the next column: from @deadlight 

That is an issue with Vanilla and resolved once we update.

> Bug: on hover the code block reveals full command line: from @kwm14 

This is also a fixed Vanilla bug which will be fixed once its updated

> Capitalise first word 'Conjure-up' to match copy doc: from @kwm14 

Sorry, the copy doc is wrong here. I'll update it.

> Padding issue on 6-col layout: by @kwm14 

This is a Vanilla issue which has been fixed and will be fixed by updating Vanilla

**Comment by pmahnke** [comment]
@anthonydillon and @frankiegranato 

some issues:

- [ ] it is missing the second/third level nav, I think it can be in it from the start
- [ ] is is missing the e-book include with 'Before you start, you’ll want this eBook'
- [ ] I think the page should end with the 'need more help' strip from the autopilot page

image

**Comment by anthonydillon** [comment]
@pmahnke 
>  is is missing the e-book include with 'Before you start, you’ll want this eBook'

I have added the ebook section to /download/cloud and the comment said we dont need it on the instructional pages.

**Comment by kwm14** [comment]
## Visual review

### Contextual footer
- [ ] Content in second block 'Growing your cloud?' doesn't match what is seen in the copy doc [here](https://docs.google.com/document/d/1b-KJ3Ext5geHtRU26UrXusLNoulSJ3jqvLl5NlwbyQo/edit#heading=h.gt89bdtcmhos)
- [ ] Links in third block 'Detailed documentation?' need removing as it doesn't match what is seen in the copy doc [here](https://docs.google.com/document/d/1b-KJ3Ext5geHtRU26UrXusLNoulSJ3jqvLl5NlwbyQo/edit#heading=h.gt89bdtcmhos)

**Comment by anthonydillon** [comment]
Thanks @kwm14 updated the copy doc and part of the contextual footer to match.

**Pull request merged**